project/source/Scripts/Objects/WeighBoat.gd

```diff
@@ -8,7 +8,7 @@ func _ready():
 func TryActIndependently():
 	pass
 
-func AddContents(new_contents):
+func AddContents(new_contents) -> bool:
```

Comment:
Do we now need to put something in the Groups documentation table that specifies this should return a `bool`?

Reply:
I don't believe so, if we opt for the change I agreed with above regarding the `active` property. Instead, we should document that on the `LabObject` page.

If we opt for that change, I think we should revert this back.